app/views/form-designer/form-home.html

```diff
@@ -0,0 +1,21 @@
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+  Form Home
+{% endblock %}
+
+{% block content %}
+
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-l">Welcome to GOV.UK Forms</h1>
+        {{ govukButton({
+        text: "Create a From",
```

Comment:
Typo:
```suggestion
        text: "Create a Form",
```

Reply:
good catch! I think this is going to haunt me throughout the project...